### PR TITLE
test: Workspace Models tests [CM-538]

### PR DESCRIPTION
**Pull request by johnkim-det**
## Ticket

CM-538

## Description

Adds automated e2e tests for "Workspace Models" section of Workspaces/Projects release party test plan

https://hpe-aiatscale.atlassian.net/wiki/spaces/ENGINEERIN/pages/1321238531/Workspaces+Projects

## Test Plan

No additional testing required, automated tests should pass.

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/e2e/models/components/ModelCreateModal.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  readonly name = new BaseComponent({
+    parent: this.body,
+    selector: '[id="modelName"]',
+  });
+  readonly description = new BaseComponent({
+    parent: this.body,
+    selector: '[id="modelDescription"]',
+  });
+  readonly addMoreDetails = new BaseComponent({
+    parent: this,
```
Comment (JComins000) [inline webui/react/src/e2e/models/components/ModelCreateModal.ts:18]: `this.body`?

Comment (JComins000) [reply webui/react/src/e2e/models/components/ModelCreateModal.ts:18]: as well as the others

**Review by JComins000 — APPROVED** [review]
honestly im just excited to see all my work being expanded upon. thanks for the contributions, very nice pace as well.

**Review by gt2345 — APPROVED** [review]
Nice example for unit tests! 
Do we think it's possible to auto generate boiler plate code such as `ModelCreateModal`

**Pull request merged**